### DISCUSSION TITLE
how can I send a slack notification when performance degradation occurs?

### DISCUSSION THREAD

**Discussion by pargevghazaryan** [ludeknovy/jtl-reporter#246] [Ideas]
how can I send a slack notification when performance degradation occurs?

**Comment by ludeknovy** [comment]
Hi @pargevghazaryan I don’t think there’s currently such a possibility.

**Reply by ludeknovy** [reply]
Is that something you would like to have there?

**Reply by pargevghazaryan** [reply]
It would be great to have a functionality to send notifications with the report link to a slack channel when degradation occurs.
